proto/substrait/algebra.proto

```diff
@@ -364,6 +388,17 @@ message AggregateRel {
     // Helps to support SUM(<c>) FILTER(WHERE...) syntax without masking opportunities for optimization
     Expression filter = 2;
   }
+
+  enum GroupingExpansion {
+    // No grouping expansion. Default.
+    GROUPING_EXPANSION_UNSPECIFIED = 0;
```

Comment:
I think in other locations NONE is specified separately from UNSPECIFICED. Can you confirm? If so, we should match that pattern here.